src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig

```diff
@@ -22,7 +22,12 @@
     <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/highlight.9.1.0.pack.js') }}"></script>
     <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/jsoneditor.min.js') }}"></script>
     <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/marked.js') }}"></script>
-    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/swagger-oauth.js') }}"></script>
+    {% if oauth2.enabled %}
+        <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/swagger-oauth.js') }}"></script>
+        <script>
```

Comment:
This isn't allowed by some CSP and we should generally avoid to inline JS for security. You can use this trick: https://github.com/Craffft/core/blob/44b89bd97280e0543dee8bd72a0a3f34cada0fa3/src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig#L34